Harvest.Web/ClientApp/src/Requests/RequestContainer.tsx

```diff
@@ -72,60 +80,78 @@ export const RequestContainer = () => {
                     setProject({ ...project, end: date as Date })
                   }
                 />
-              </FormGroup>
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <FormGroup tag="fieldset">
-                <Label>Which type of crop will we grow?</Label>
-                <Crops
-                  crops={project.crop}
-                  setCrops={(c) => setProject({ ...project, crop: c })}
-                ></Crops>
-              </FormGroup>
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <FormGroup>
-                <Label>Who will be the PI?</Label>
-                <SearchPerson
-                  user={project.principalInvestigator}
-                  setUser={(u) =>
-                    setProject({ ...project, principalInvestigator: u })
-                  }
-                ></SearchPerson>
-              </FormGroup>
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <FormGroup>
-                <Label>What are the requirements?</Label>
-                <Input
-                  type="textarea"
-                  name="text"
-                  id="requirements"
-                  value={project.requirements}
-                  onChange={(e) =>
-                    setProject({ ...project, requirements: e.target.value })
-                  }
-                  placeholder="Enter a full description of your requirements"
-                />
-              </FormGroup>
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <Button color="primary" onClick={create}>
-                Create Field Request
-              </Button>
-            </Col>
-          </Row>
-        </Container>
+              </div>
+            </FormGroup>
+          </div>
+        </div>
+        <FormGroup>
+          <Label>Which type of crop will we grow?</Label>
+          <div className="custom-control custom-radio">
+            <input
+              type="radio"
+              id="customRadio1"
+              name="customRadio"
+              className="custom-control-input"
+              style={{ zIndex: 1 }} //prevent class custom-control-input from blocking mouse clicks
+              value="Row"
+              checked={project.cropType === undefined || project.cropType === "Row"}
```

Comment:
@sprucely could be nice to just default it to row and make it not able to be undefined.

Reply:
@sprucely looking at the code i don't really see how cropType could be undefined since it's basically a two value enum?

Reply:
The interface/type doesn't allow undefined, but state was initialized from an almost empty object. I'll just make the initial state include `cropType: "Row"`